packages/select/src/ResourceSelect.js

```diff
@@ -173,18 +175,22 @@ const ResourceSelect = ({
     } else {
       fetch = () =>
         resource.postGet(
-          qs.stringify(params, {
-            encode: false,
-            arrayFormat: 'repeat',
-            indices: false,
-            allowDots: true,
-          }),
+          qs.stringify(
+            { ...params },
```

Comment:
i think this should probably go back to how it was before without the unnecessary spread